esterni/piano-di-progetto/components/pianificazione.tex

```diff
@@ -583,7 +584,7 @@
 ~~\label{fig:gantt_incremento_4}
 \end{figure}
 % sub:incremento_4 (end)
-\subsection[Incremento 5]{Incremento 5 {\normalsize\normalfont\itshape(14/04/2020 \symbol{8594} 20/04/2020)}}%
+\subsection[Incremento 5]{Incremento 5 {\normalsize\normalfont\itshape(05/05/2020 \symbol{8594} 11/05/2020)}}%
```

Comment:
La presentazione da preparare per la revisione di qualifica non è da realizzare nel periodo che va dal 11/05 al 18/05?

Reply:
si sto lavorando proprio adesso in locale per aggiungere alla pianificazione la sezione di preparazione alla rq e correggere tutti i gantt

Reply:
Quindi, io lo metterei in preparazione in entrata alla RQ(che è ancora da fare) e lo toglierei dall'incremento come negli altri incrementi